lib/license_finder/package_managers/glide.rb

```diff
@@ -35,7 +35,10 @@ def package_management_command
     end
 
     def prepare_command
-      'glide install'
+      command = 'glide install'
+      command = "cd src; #{command}; cd .." if File.exist?(project_path.join('src', 'glide.lock'))
```

Comment:
@xtreme-jason-smith Is it possible that both `project_path` and `project_path/src` may have the `glide.lock` file? If it is possible, how do we want to handle that?

If we know for sure that it never happens, feel free to Resolve this comment. The rest looks good to me

Reply:
Yea i'm not sure if this is a safe assumption to make. The proper way to do this would be to make sure the --project_path argument is being used properly. It should install the glide command using that path. We really shouldn't be changing directories in the prepare commands.

Reply:
@xtreme-shane-lattanzio - Yes, that would make sense. However, the glide package manager checks both the root folder and the `src` sub folder to determine if it's an active glide project. So, to do your suggestion, we would need to remove the `src` folder check, which would revert the change where certain projects put their glide lock files in the `src` folder instead of the root project.

I guess we could ask tracker to fix their glide project to host those files in the root folder instead?  However, I wonder if this is common practice for glide, meaning the `glide.lock` file are not always in the base directory.

Reply:
@pivotal-pmital We could run it in both folders if we find the lock file in both. Though like you I'm not sure how common that would be.

Also, as mentioned in reply to Shane, is the team that has this issue created their glide project correct? Maybe the glide.lock file should not be in the `src` folder, but in the base folder.